rke/templates/templates.go

```diff
@@ -156,10 +157,11 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.16.0-alpha": kubeDnsv18,
 		},
 		kdm.MetricsServer: {
-			">=1.23.3-rancher1-1":                     metricsServerv061,
-			">=1.20.14-rancher2-1 <1.23.3-rancher1-1": metricsServerv050,
-			">=1.20.4-rancher1-1 <1.20.14-rancher2-1": metricsServerv120,
-			">=1.8.0-rancher0 <1.20.4-rancher1-1":     metricsServerv18,
+			">=1.23.3-rancher1-1":                       metricsServerv061,
+			">=1.22.15-rancher1-1 <1.23.3-rancher1-1":   metricsServerv051fix,
+			">=1.20.14-rancher2-1 <=1.22.13-rancher1-1": metricsServerv050,
```

Comment:
I've just realized that all other template version ranges use exclusive top end of range, so _ideally_ we want to be consistent and have it as below:
```suggestion
			">=1.20.14-rancher2-1 <1.22.15-rancher1-1": metricsServerv050,
```

However this is a nitpick plus I was the one who suggested inclusive top end of the range, so approving.

If you have a chance to fix it before the second approval is done, that would be great. If this is the only change, consider my approval still valid even if GitHub automatically removes it.

Reply:
I've updated your suggestion.